pytket/extensions/cutensornet/backends/cutensornet_backend.py

```diff
@@ -232,25 +233,43 @@ def get_operator_expectation_value(
         self,
         state_circuit: Circuit,
         operator: QubitPauliOperator,
+        post_selection: Optional[Union[None, dict[Qubit, int]]] = None,
```

Comment:
Why is it `Optional` **and** `Union[None, ...]`? Isn't `Optional` precisely the same as `Union[None, ...]`?